compute/disks/disks_test.go

```diff
@@ -204,6 +255,41 @@ func TestComputeDisksSnippets(t *testing.T) {
 		}
 	})
 
+	t.Run("Create and resize a regional disk", func(t *testing.T) {
```

Comment:
Do these tests need to be in the same function as all the others, or could this be done in a more focused (and maintainable!)  `TestRegionalDiskSamples()` method?

Reply:
That's fair. I think these tests would benefit from some refactoring, but that's beyond the scope of what you're doing here.